etc/profile-a-l/dino-im.profile

```diff
@@ -1,11 +1,10 @@
 # Firejail profile for dino-im
 # Description: Modern XMPP Chat Client using GTK+/Vala, Ubuntu specific bin name
 # This file is overwritten after every install/update
-##quiet
 # Persistent local customizations
 include dino-im.local
 # Persistent global definitions
-include globals.local
+#include globals.local
```

Comment:
Add the below line just above '#include globals.local' please,  to make it consistent with our other profiles
`# added by included profile`

Reply:
Oh sorry, missed that, added it now.